src/NServiceBus.Core/Recoverability/Recoverability.cs

```diff
@@ -86,9 +87,11 @@ protected internal override void Setup(FeatureConfigurationContext context)
                     policy = DefaultRecoverabilityPolicy.Invoke;
                 }
 
+                var failedConfig = new FailedConfig(errorQueue, context.Settings.Get<HashSet<Type>>(UnrecoverableExceptions));
```

Comment:
Why is this looking up the UnrecoverableExceptions by type from the settings instead of using the `Recoverability.UnrecoverableExceptions` used in the setting methods?